task.go

```diff
@@ -41,6 +42,14 @@ type Status struct {
 	ExitTime time.Time
 }
 
+type ProcessInfo struct {
+	// Pid is the process ID
+	Pid uint32
+	// Info includes additional process information
+	// Info varies by platform
+	Info *google_protobuf.Any
```

Comment:
If you can convert this to the runtime specfic type, I would just use an interface here. That is much easier for clients to use. If you can't convert, it is okay to store an `Any` in an `interface{}`.

Reply:
@crosbymichael suggested I make `Info` an `Any` unless I did it wrong?

Reply:
Yes, Did this do that its a lazy unmarshal and we don't have to import platform specific types in the client if they are not used.  The imports stay with the consumer.

Reply:
We do the same on containers.Container

Reply:
@crosbymichael I gave this feedback in the interest of avoiding exposure of the GRPC types on the API directly. `Any` is definitely an exception there.

Let's leave as is for now and we can re-examine this later.